addons/website/models/website.py

```diff
@@ -675,8 +675,98 @@ def configurator_apply(self, **kwargs):
         IrQweb = self.env['ir.qweb'].with_context(website_id=website.id, lang=website.default_lang_id.code)
         snippets_cache = {}
         translated_content = {}
+        fonts_attributes = {}
+        html_storage = {}
+
+        def _replace_tags_with_placeholders(html_string):
```

Comment:
Maybe not a good name given that we currently use "placeholder" for the whole terms we send to IAP :grimacing: Maybe just `_transform_tags` would be enough?